libs/ledger-live-common/src/derivation.ts

```diff
@@ -156,16 +156,14 @@ const modes = Object.freeze({
   polkadotbip44: {
     overridesDerivation: "44'/354'/<account>'/0'/<address>'",
   },
-  // glif legacy derivation
-  gliflegacy: {
```

Comment:
if filecoin is already in production, i think we can't afford changing the derivation id, it would crash existing accounts.